Cookbook/Technical-Documents/OutOffOfficeRequest.md

```diff
@@ -0,0 +1,61 @@
+# Requesting and Notifying Out Of Office time
+
+Someone who is Out Of Office means that is not in their usual work place at their usual working hours. This can be when someone goes on holidays or when someone that usually works in the office is working from somewhere else.
```

Comment:
``
```suggestion
Someone who is Out Of Office means that they are not in their usual work place at their usual working hours. This can be when someone goes on holidays or when someone that usually works in the office is working from somewhere else.
```